pkg/server/registry/pod_registry.go

```diff
@@ -212,6 +212,13 @@ func (reg *PodRegistry) UpdatePodSpecAndLabels(p *api.Pod) (*api.Pod, error) {
 		return nil, err
 	}
 	p, err = reg.AtomicUpdate(p.Name, func(in *api.Pod) error {
+		if api.IsTerminalPodPhase(in.Spec.Phase) {
+			return fmt.Errorf("pod is in a terminal pod phase")
```

Comment:
one small change: can we have this say 'pod **spec** is in a terminal pod phase".  It's a minor difference but a bit clearer since we have 2 phases and the wording "is in a terminal pod phase" makes it sound like it's referring to the status of the pod

Reply:
Sounds good :+1: